Tchap/Modules/Settings/SettingsViewController.m

```diff
@@ -101,6 +101,7 @@
 
 enum {
     CRYPTOGRAPHY_INFO_INDEX = 0,
+    CRYPTOGRAPHY_IMPORT_INDEX,
```

Comment:
Please display the Export option first (like in Tchap-Android)
This new option "Import" is expected below the "Export"